citadel/install_ubuntu_src.md

```diff
@@ -62,7 +62,10 @@ An alternative method is to use the `.deb` packages available on Debian or Ubunt
 
 ```bash
 sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
-sudo -E apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+
+sudo apt-get install curl # if curl isn't already installed
```

Comment:
How about moving this to line 27?

Reply:
8603816d92c17c1b684f2c8d2677122385e61788